gazebo_arm_controller:
  type: position_controllers/JointGroupPositionController
  joints:
    - joint_a
    - joint_b
    - joint_c
    - joint_d
    - joint_e
    - joint_f
  gains:
    joint_a:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_b:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_c:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_d:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_e:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    joint_f:
      p: 100
      d: 1
      i: 1
      i_clamp: 1